src/AppBundle/Entity/Offer.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use AppBundle\Entity\Activity;
+use AppBundle\Entity\User;
+
+/**
+ * Offer
+ *
+ * @ORM\Table(name="offers")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\OfferRepository")
+ */
+class Offer
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Activity", inversedBy="offers")
+     * @ORM\JoinColumn(name="activity_id", referencedColumnName="id")
+     */
+    protected $activity;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="User", inversedBy="offers")
+     * @ORM\JoinColumn(name="user_id", referencedColumnName="id")
+     */
+    protected $user;
+
+    /**
+     * @var float
+     *
+     * @ORM\Column(name="price", type="float")
+     */
+    private $price;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="childrenAgeFrom", type="integer")
```

Comment:
I don't think we need 'children' prefix since it's quite obvious what we are talking about :) it will be kinda overhead to write this every time